tests/test_ci_benchmarks.py

```diff
@@ -0,0 +1,15 @@
+"""
+This file has benchmarks used in CI to catch regressions.
+
+This is in addition to the performance benchmarking in `../../benchmarks`, which track
+the speed of functions over time.
+"""
+
+import pytest
+
+from xdsl.dialects.arith import ConstantOp
+
+
+@pytest.mark.benchmark
+def test_const_verify():
+    ConstantOp.from_int_and_width(1, 32).verify()
```

Comment:
Can we not reuse some of the `xdsl-bench` stuff, instead of basing performance regressions on a test which makes and verifies one operation?

Reply:
Yep definitely would be nice to keep them coupled in some way, I first wanted to see whether this would work at all.